libgalois/include/katana/PropertyGraph.h

```diff
@@ -237,6 +237,43 @@ class KATANA_EXPORT PropertyGraph {
     rdg_.set_view_name(fmt::format("rdg-{}-part{}", part_policy, num_hosts));
   }
 
+  const std::unordered_map<tsuba::EntityTypeID, std::vector<tsuba::EntityTypeID>>& node_entity_type_id_dictionary()
```

Comment:
Why do we need this getter?

Reply:
OK, I see that it's private. So that's good. We need a better name perhaps to signal that this is the info coming from RDG/Storage layer.

Reply:
@e-mcginnis